lib/common/patchset.c

```diff
@@ -298,11 +298,6 @@ patchset_process_digest(xmlNode *patch, xmlNode *source, xmlNode *target,
      */
     CRM_LOG_ASSERT(!xml_document_dirty(target));
 
-    crm_element_value_int(patch, PCMK_XA_FORMAT, &format);
-    if ((format > 1) && !with_digest) {
-        return;
-    }
```

Comment:
I didn't because this function didn't previously validate the format. All it did was say "ignore the `with_digest` argument if `format == 1`." Otherwise we don't do anything differently based on format.